sydent/http/servlets/infoservlet.py

```diff
@@ -87,15 +86,20 @@ def render_GET(self, request):
 
         result = copy.deepcopy(result)
 
+        # Remove 'requires_invite' if found
+        if 'requires_invite' in result:
+            result.pop('requires_invite')
+
         if self.sydent.nonshadow_ips:
             ip = IPAddress(self.sydent.ip_from_request(request))
+
+            # Remove shadow_hs from response if user is from a shadow server
```

Comment:
we aren't just removing shadow_hs? "Move shadow_hs to hs" would be more accurate

(also, aiui this is for use from clients, not servers?)

Reply:
Synapse doesn't even use `shadow_hs`, so going to remove from response.

Reply:
right, but doesn't synapse use `hs` ?

Reply:
It does, to check that the user is allowed to register.